src/StaticWebAssetsSdk/Tasks/GenerateStaticWebAssetEndpointsManifest.cs

```diff
@@ -137,6 +139,48 @@ public override bool Execute()
         return !Log.HasLoggedErrors;
     }
 
+    private static void EncodeLinkHeadersIfNeeded(StaticWebAssetEndpoint endpoint)
+    {
+        for (var i = 0; i < endpoint.ResponseHeaders.Length; i++)
+        {
+            ref var header = ref endpoint.ResponseHeaders[i];
+            if (!string.Equals(header.Name, "Link", StringComparison.OrdinalIgnoreCase))
+            {
+                continue;
+            }
+            var headerValues = header.Value.Split([','], StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Why passing `[','` and not jut `','`?

Reply:
This is an msbuild task that also targets framework